augur/application/cli/db.py

```diff
@@ -66,7 +66,6 @@ def add_repos(filename):
                 
                 print(
                     f"Inserting repo with Git URL `{repo_data['url']}` into repo group {repo_data['repo_group_id']}")
-                controller.add_cli_repo(repo_data)
```

Comment:
It appears that controller.add_cli_repo has been removed

Reply:
Oh that was a typo when I was removing a test print statement before making the PR. My bad